eng/Versions.props

```diff
@@ -32,6 +32,7 @@
     <SystemSecurityCryptographyCngPackageVersion>4.6.0-preview.19105.1</SystemSecurityCryptographyCngPackageVersion>
     <SystemSecurityCryptographyXmlPackageVersion>4.6.0-preview.19105.1</SystemSecurityCryptographyXmlPackageVersion>
     <SystemTextEncodingsWebPackageVersion>4.6.0-preview.19105.1</SystemTextEncodingsWebPackageVersion>
+    <MicrosoftDotNetGenApiPackageVersion>1.0.0-beta.19081.1</MicrosoftDotNetGenApiPackageVersion>
```

Comment:
```suggestion
	<!-- Packages from dotnet/arcade -->
    <MicrosoftDotNetGenApiPackageVersion>1.0.0-beta.19081.1</MicrosoftDotNetGenApiPackageVersion>
```

Can you also add this to Version.Details.xml under ToolsetDependencies?

Reply:
Done